requirements/requirements.txt

```diff
@@ -1,3 +1,4 @@
+pillow!=8.4.0 # Fails to build
 deprecation
 numpy>=1.13
```

Comment:
`pillow<8.4; ; python_version < "3.7"` perhaps?

Need also in case update `pyproject.toml` (and ideally also other requirements* files

Reply:
I have added pillow<8.4 to all places where matplotlib is required as it is a transient dependency of matplotlib.